extensions/signal.sigmf-ext.md

```diff
@@ -0,0 +1,285 @@
+Metadata Format (SigMF) specification. This extension namespace defines how to
+describe the attributes of wireless communications systems.
+
+## Copyright Notice
+
+This document is Copyright of The GNU Radio Foundation, Inc.
+
+This document is available under the [CC-BY-SA License](http://creativecommons.org/licenses/by-sa/4.0/).
+
+<a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png" /></a>
+
+## Conventions Used in this Document
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”,
+“SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be
+interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+JSON keywords are used as defined in [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf).
+
+Augmented Backus-Naur form (ABNF) is used as defined by [RFC
+5234](https://tools.ietf.org/html/rfc5234) and updated by [RFC
+7405](https://tools.ietf.org/html/rfc7405).
+
+Fields defined as "human-readable", a "string", or simply as "text" shall be
+treated as plaintext where whitespace is significant, unless otherwise
+specified.
+
+## Specification
+
+This specification defines an extension `signal` namespace
+that can be used in SigMF `annotations` to describe wireless communications
+systems and their emitters.
+
+### Annotations
+
+This extension adds the following field to the `annotations` global SigMF object:
+
+|name|required|type|description|
+|----|--------------|-------|-----------|
+|`emission`|false|object|Describes the communications system of a signal emission and its emitter.|
+
+The field of communications is vast, and there may be communications systems that cannot be described using the names and fields described in this extension. If you need additional or different fields to describe a system, create a new extension that adds the necessary fields to the `signal` namespace and/or submit the new fields to be upstreamed into this specification.
+
+#### The `emission` Object
+
+`emission` objects contain name/value pairs that describe the modulations of communications systems. These objects MAY contain the following pairs:
+
+|name|required|type|description|
+|----|--------------|-------|----------|
+|`label`|false|string|e.g., wifi|
+|`detail`|false|[Detail](signal.sigmf-ext.md#the-detail-object)|Emission details (standard, modulation, etc.)|
+|`emitter`|false|[Emitter](signal.sigmf-ext.md#the-emitter-object)|Emitter details (manufacturer, geo coordinates, etc.)|
+
+##### The Emitter Object
+
+|name|required|type|description|
+|----|--------|----|-----------|
+|`seid`|false|uint|Unique ID of the emitter|
+|`manufacturer`|false|string|Manufacterer of the hardware used to emit the signal|
+|`latitude`|false|float|Latitude|
+|`longitude`|false|float|longitude|
```

Comment:
We should probably follow the NTIA's example, here, and specify the units as "decimal degrees" for both long & lat. Near the top of @MichaelCotton's comment, [here](https://github.com/gnuradio/SigMF/pull/87#issuecomment-508749532)